Frontend/src/pages/marketplace/Marketplace.jsx

```
@@ -278,337 +369,48 @@ function App() {
-                          <path d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z" />
-                        </svg>
-                      ))}
+                    <div className="mt-2 flex items-center gap-2">
+                      <div className="flex">
+                        {[...Array(5)].map((_, i) => (
+                          <svg
+                            key={`star-${product.id}-${i}`} // Optional: Unique key for stars
+                            className={`h-3 w-3 sm:h-4 sm:w-4 ${i < product.rating ? "text-yellow-400 fill-yellow-400" : "text-gray-300 fill-gray-300"}`}
+                            xmlns="http://www.w3.org/2000/svg"
+                            viewBox="0 0 24 24"
+                          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)